app/models.py

```diff
@@ -23,7 +23,7 @@ class Todo(Base):
     title = Column(String, index=True, nullable=False)
     details = Column(String, nullable=True)
     completed = Column(Boolean, default=False, nullable=False)
-    created_at = Column(DateTime, default=datetime.now(timezone.utc), nullable=False)
+    created_at = Column(DateTime(timezone=True), server_default=func.now(), nullable=False)
     list_id = Column(Integer, ForeignKey("lists.id"))
```

Comment:
```suggestion
    list_id = Column(Integer, ForeignKey("lists.id", ondelete="CASCADE"))
```
Just a suggestion, depending on how you want to handle it. Like this if a list gets deleted and the list.id is no longer available, all related todos would also be removed.